api/src/controllers/zot4planimport.ts

```diff
@@ -168,9 +197,14 @@ const zot4PlanImportRouter = router({
     .input(z.object({ scheduleName: z.string(), studentYear: z.string() }))
     .query(async ({ input, ctx }) => {
       const originalScheduleRaw = await getFromZot4Plan(input.scheduleName);
-      const res = convertIntoSavedRoadmap(originalScheduleRaw, input.scheduleName, getStartYear(input.studentYear));
+      const savedRoadmap = convertIntoSavedRoadmap(
+        originalScheduleRaw,
+        input.scheduleName,
+        getStartYear(input.studentYear),
+      );
+      const apExams = await getApExamsFromZot4Plan(originalScheduleRaw);
       await db.insert(zot4PlanImports).values({ scheduleId: input.scheduleName, userId: ctx.session.userId });
-      return res;
+      return [savedRoadmap, apExams] as [SavedRoadmap, userAPExam[]];
```

Comment:
In React/TS, multiple return items are usually done in an object as opposed to an array.
```suggestion
      return { savedRoadmap, apExams }
```
The type should also be inferred correctly when it's done this way